src/convert.jl

```diff
@@ -1026,3 +1026,57 @@ function convert(::Type{HParallelotope}, Z::AbstractZonotope{N}) where {N}
     end
     return HParallelotope(D, c)
 end
+
+"""
+    convert(::Type{Zonotope}, cp::CartesianProduct{N, <:AbstractZonotope{N}
+            , <:AbstractZonotope{N}}) where {N}
```

Comment:
```suggestion
    convert(::Type{Zonotope}, cp::CartesianProduct{N, AZ1, AZ2}) where
         {N, AZ1<:AbstractZonotope{N}, AZ2<:AbstractZonotope{N}}
```
This is to match the signature in the docs and also to generalize it to two different types of zonotopes (the code is general).